dask_cassandra_loader/loader.py

```diff
@@ -650,37 +640,51 @@ def load_cassandra_table(self, table_name, projections, and_predicates,
         :param partitions_to_load: List of tuples. Each tuple as a column name as String.
          and a list of keys which should be selected. It should only contain columns which are partition columns.
         :param force: It is a boolean. In case all the partitions need to be loaded, which is not recommended,
-         it should be set to 'True'.
+         it should be set to 'True'. By Default it is set to 'False'.
+        :param debug: It is a boolean. If set to True it prints to the logs the query to be sent to Cassandra. By
+         default it is set to 'False'.
         """
 
         table = Table(self.cassandra_con.keyspace, table_name)
 
-        table.load_metadata(self.cassandra_con)
-        if table.error:
-            raise Exception("load_cassandra_table failed: " + table.error)
+        try:
+            table.load_metadata(self.cassandra_con)
+        except Exception as e:
+            raise Exception("load_cassandra_table failed: " + str(e))
```

Comment:
Can you create a custom exception like 

```python
class DaskCassandraLoaderException(Exception):
  pass
```
Than it easier to catch for it specifically.

Also you are swallowing information of the caught exception. The parent exception can be retained by chaining like
```python
raise Exception("load_cassandra_table failed: " + str(e)) from e
```

Reply:
What is the difference between:
```
except Exception as e:
   raise Exception("load_cassandra_table failed: " + str(e))
```
and
```
except Exception as e:
    raise Exception("load_cassandra_table failed: " + str(e)) from e
```

Reply:
@sverhoeven should I create one exception class for DaskCassandraLoader or also for the LoadingQuery and Table? Since I am creating for one, I could create for all.

Reply:
I think one is enough. For the rest the Exception message bring the the function name with it. This was what normally we would do when implement cascade Exceptions in database systems when processing a query.

Covered by 0353eab

Reply:
`from e` gives you a cause attribute on the exception, see https://www.python.org/dev/peps/pep-3134/.

Yes, one exception class is enough.

Reply:
But then I do not need the `str(e)` right?